pkg/deployment/all-in-one.go

```diff
@@ -39,10 +40,14 @@ func (a *AllInOne) Get() *appsv1.Deployment {
 	labels := a.labels()
 	trueVar := true
 
+	args := append(a.jaeger.Spec.AllInOne.Options.ToArgs())
+
+	adminPort := util.GetPort("--admin-http-port=", args, 14269)
```

Comment:
Could you please explain what does 14269? Is it the dafault?

Reply:
yes it is the default - so it is just checking for the default being overridden by an arg.